### ISSUE TITLE
0.9 Procedures

### ISSUE THREAD

**Issue by sole-practitioner** [freemed/freemed#14]
I have populated database with data for CPT, Facility, ICD, Insurance Co, Place of Serv, Practice, Provider,Type of Serv, then made a patient, tried to add a procedure.  Seemed to work OK, got a pop-up confirming fee for procudure, got a blip on bottom right that procedure was added, but it took me back to list, and no procedure is in the list. Tried choosing coverage on next tab, same result - right back to empty list.

I set up the VM version as a test, same thing.  What might I be missing?  Thanks for your help!  -tbh

**Comment by sole-practitioner** [comment]
Is there something missing in the installation instructions for Remitt setup?  The instructions end after building Remitt.  But it appears the Freemed is looking for Remitt on port 8080, and apache isn't listening on port 8080.
There is a "Readme" that explains a bunch of things to set up Remitt involving Tomcat, is that what needs to be done?

**Comment by sole-practitioner** [comment]
Tried installing Tomcat, adding the "-Dremitt..." to the configuration file and symbolic link of logs (all of this was done on the VMWare version).  This managed to get a response on port 8080, but still no luck getting procedures to work.

**Comment by jbuchbinder** [comment]
Yes, REMITT does need to be running to use any of the billing processing bits. Is your tomcat log indicating any issues bringing REMITT up?
